lib/plurimath/math/function/multiscript.rb

```diff
@@ -78,6 +84,32 @@ def prescripts
         def valid_value_exist?(field)
           !field && (field&.empty? || field&.all?(None))
         end
+
+        def unicode_valid_value?(field)
+          !(field.empty? && valid_value_exist?(field))
```

Comment:
@suleman-uzair I think this condition will always return true, can you verify if this.